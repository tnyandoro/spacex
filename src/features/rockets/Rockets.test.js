import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import configureStore from 'redux-mock-store';
import Rockets from './Rockets';
import reducer from './rocketsReducer';

describe('Missions Component', () => {
  const initialState = {
    rocketsReducer: [],
    missionsReducer: [],
  };
  const mockStore = configureStore();
  const store = mockStore(initialState);

  it('Rockets snapshot test', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <Rockets />
        </Provider>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});

describe('mission reducer', () => {
  it('Shoud return initial state', () => {
    expect(reducer([], [])).toEqual([]);
  });
});
