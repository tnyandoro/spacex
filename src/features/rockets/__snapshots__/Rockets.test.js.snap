// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Missions Component Rockets snapshot test 1`] = `
<div
  className="rocket-wrapper"
>
  <ul
    className="ul-rocket"
  />
</div>
`;
